### PR TITLE
Update ExoticaDQM_cfi.py

### PR DESCRIPTION
**Pull request by aescalante**
Modified list of valid HLT trigger path names due to a typo in one of the trigger names.

### CODE REVIEW

**Comment by deguio** [comment]
please submit to 75x and 76x as well.
wait to see the PR there before approving.
thanks,
F.

**Comment by aescalante** [comment]
Hello, 

Do I have to create a new PR in 75X and 76X or is possible to propagate this request to the other versions?.
thanks.
Alberto.

**Comment by deguio** [comment]
you have to create them manually..  :(
you can cherry-pick the commit on top of 75 and 76 branches.
ciao,
F.

**Comment by deguio** [comment]
@aescalante 
I see in the selections the following:
"HLT_"

is this wanted?
F.

**Comment by aescalante** [comment]
In principle I am interested only the list of triggers  trigger names before the last one, HLT_ , but for the beginning of the run I don’t want to see empty histograms due to a problem with the trigger names so for the moment I prefer to leave it like this and remove HLT_ selection in the future. 

Other point is that there is a new RelVal sample for displaced fermions and there is an intention to add dedicated histograms to the exotica module, but before that in other to get filled the current histograms in RelMon for that relval I prefer to leave temporary the "HLT_ " until the new histograms are added and a HLT name is decided for those events.

In conclusion I agree that in long term this is not desired, but for the moment I prefer to have it. Of course, if there is is a problem with it I can modify the commit and remove the HLT_ because the main purpose of this commit was a fix in the Ele trigger name.

I hope is clearer now.
Alberto.

**Comment by deguio** [comment]
+1

**Comment by perrotta** [comment]
please test

**Comment by davidlange6** [comment]
+1
